jmeter-service/jmeterUtils.go

```diff
@@ -211,11 +211,10 @@ func execute(testInfo TestInfo, workload *Workload, err error, jMeterCommandLine
 	}
 	// Step 3: Parse result and lets analyze the result
 	result, err := parseJMeterResult(jmeterCommandResult, testInfo, workload, funcValidation)
-	if result && err != nil {
-		logger.Debugf("Successfully executed JMeter test: %v", testInfo)
-	} else {
-		logger.Errorf("Could not parse JMeter test (%v). Error: %s  ", testInfo, err.Error())
+	if err != nil {
+		logger.Errorf("Could not parse JMeter test (%v). Error: %v  ", testInfo, err)
 	}
+	logger.Debugf("Jmeter tests passed: %t. TestInfo: %v", result, testInfo)
```

Comment:
There's no return after the error message so I think we have a situation here where both error and success message could be printed at the same time